crates/qasm2/Cargo.toml

```diff
@@ -10,6 +10,7 @@ name = "qiskit_qasm2"
 doctest = false
 
 [dependencies]
+num-bigint = "0.4"
```

Comment:
Should we move this to the workspace, because we use big int in accelerate too.

Reply:
Sure, done in 27e1689c6a.